src/Components/Search/SearchBar.tsx

```diff
@@ -244,7 +245,7 @@ export class SearchBar extends Component<Props, State> {
     ) => ({
       action_type: Schema.ActionType.SelectedItemFromSearch,
       query: state.term,
-      destination_path: href,
+      destination_path: displayType === "Artist" ? `${href}/works-for-sale` : href,
```

Comment:
Sorry, I didn't mean to just update this. The suggestion made that a little unclear, my bad!

We can use `displayType` to build the url _instead_ of using `regex`. So we'd want to update [line 33](https://github.com/artsy/reaction/pull/3351/files#diff-195cf326e4088c5f9ff2c6d47cc4248cR33) and [line 337](https://github.com/artsy/reaction/pull/3351/files#diff-195cf326e4088c5f9ff2c6d47cc4248cR337)

Reply:
My bad! I used that github automatic "commit this suggestion" thing

Reply:
You did, I just shouldn't have used GitHub's suggestion for that because it wasn't actually the right place to make a code change 😅